ansible/group_vars/all/telemetry_certs.yml

```diff
@@ -0,0 +1,12 @@
+# Configure telemetry server and dsmsroot key,cer
+
+telemetry_certs:
+    server_key: "/etc/sonic/telemetry/streamingtelemetryserver.key"
+    server_csr: "/etc/sonic/telemetry/streamingtelemetryserver.csr"
+    server_cer: "/etc/sonic/telemetry/streamingtelemetryserver.cer"
+    dsmsroot_key: "/etc/sonic/telemetry/dsmsroot.key"
+    dsmsroot_csr: "/etc/sonic/telemetry/dsmsroot.csr"
+    dsmsroot_cer: "/etc/sonic/telemetry/dsmsroot.cer"
+    client_key: "/etc/sonic/telemetry/streamingtelemetryclient.key"
+    client_cer: "/etc/sonic/telemetry/streamingtelemetryclient.cer"
```

Comment:
Not seeing client_key is being used? how the client cert is being copied to this location on ptfhost?

Reply:
yes

Reply:
yes, this will be used on the client-side. But as I mentioned copy on ptfhost doesn't work. Currently in the code I have tried to copy certs. Once certs are copied I can rename accordingly.